### PR TITLE
(registry) feat: add donation link

### PR DESCRIPTION
**Pull request by mohandast52**
## Proposed changes

- Donation link added

Screensh2

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: apps/autonolas-registry/common-util/Details/DetailsSubInfo/RewardsSection.tsx [hunk 1]
```diff
@@ -19,17 +22,22 @@ const RewardColumn = ({ title, statistic }: { title: string; statistic: string }
...
-    </Row>
+    <Flex vertical gap={4}>
+      <Row>
+        <RewardColumn title={'Claimable Reward'} statistic={`${rewardsFormatter(reward, 4)} ETH`} />
+        <RewardColumn title={'Claimable Top-Up'} statistic={`${rewardsFormatter(topUp, 2)} OLAS`} />
+      </Row>
+      <Row>
+        <Link href="https://tokenomics.olas.network/donate">
+          Make donation {UNICODE_SYMBOLS.EXTERNAL_LINK}
+        </Link>
```
Comment (truemiller) [inline apps/autonolas-registry/common-util/Details/DetailsSubInfo/RewardsSection.tsx:39]:
```suggestion
        <a href="https://tokenomics.olas.network/donate">
          Make donation {UNICODE_SYMBOLS.EXTERNAL_LINK}
        </a>
```

**Review by truemiller — APPROVED** [review]
tiny suggestion, but approved

File: apps/autonolas-registry/common-util/Details/DetailsSubInfo/RewardsSection.tsx (hunk 1 above)
Comment (Tanya-atatakai) [reply apps/autonolas-registry/common-util/Details/DetailsSubInfo/RewardsSection.tsx:39]: shouldn't it also have `target="_blank"`? @mohandast52

**Review by oaksprout — APPROVED** [review]

File: apps/autonolas-registry/common-util/Details/DetailsSubInfo/RewardsSection.tsx (hunk 1 above)
Comment (Tanya-atatakai) [reply apps/autonolas-registry/common-util/Details/DetailsSubInfo/RewardsSection.tsx]: hmm, we open a new tab here when click "guide" https://tokenomics.olas.network/bonding-products

and why to have the `EXTERNAL_LINK` symbol then?

**Pull request merged**